atom/browser/native_window_views_win.cc

```diff
@@ -72,6 +72,12 @@ const char* AppCommandToString(int command_id) {
   }
 }
 
+bool IsScreenReaderActive() {
```

Comment:
This block courtesy of https://github.com/electron/electron/commit/9c9ba809783b838cd1ff06c974bb47e2cfc40e9d